yarn.lock

```diff
@@ -3093,6 +3093,15 @@
     nanoid "3.3.4"
     query-string "^7.1.1"
 
+"@mintbase-js/wallet@0.5.0-beta.0":
```

Comment:
Can you run `yarn` to pick up the latest version of `@mintbase-js/wallet`?

Reply:
done